src/kixi/datastore/schemaextracter.clj

```diff
@@ -17,8 +19,7 @@
 (defn metadata->file
   [filestore metadata]
   (let [id (get-in metadata [:id])
-        f (File/createTempFile id ".tmp")]
-    (.deleteOnExit f)
+        f (temp-file id ".tmp")]
```

Comment:
yes, yes i am :)

No tests for this NS at the mo, it's about to be gutted and transformed into something entirely different, before you react in horror!